test_defopt.py

```diff
@@ -361,9 +374,21 @@ def test_enumgetter(self):
                          msg='argparse needs to report this value')
 
 
-class Choice(Enum):
-    one = 1
-    two = 2
+class TestTuple(unittest.TestCase):
+    def test_tuple(self):
+        def main(foo):
+            return foo
+        main.__annotations__ = {'foo': typing.Tuple[int, str]}
```

Comment:
This will work in the docstring too, right? I'd prefer to test it that way since it's the more realistic way people will use this in Python 2. You could use both at once and I think the code will ensure they are the same.